config/config.go

```diff
@@ -22,7 +22,7 @@ const (
 	// SettingListen is the config key for the listen address
 	SettingListen = "listen"
 	// SettingListenDefault is the default value for the listen address
-	SettingListenDefault = ":8080"
+	SettingListenDefault = ":11080"
```

Comment:
Was this committed by mistake?

Reply:
oh crap, yes, I committed my local changes. sorry about that. good catch.